File: CS/SpreadWinDemoCS/FirstViewPage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;

namespace SpreadWinDemo
{
    public partial class FirstViewPage : UserControl
    {
        public FirstViewPage()
        {
            InitializeComponent();

            this.Load += new System.EventHandler(this.FirstViewPage_Load);
        }

        private void FirstViewPage_Load(object sender, EventArgs e)
        {
            // ラベルの背景をPictureBoxコントロールに対して透過にする
            mainvisual.Controls.Add(product1);
            mainvisual.Controls.Add(product2);
            mainvisual.Controls.Add(product3);
            mainvisual.Controls.Add(product_body);
            mainvisual.Controls.Add(product_title);
        }
    }
}

File: VB/SpreadWinDemo/FirstViewPage.vb
Public Class FirstViewPage

    Public Sub New()

        ' この呼び出しは、Windows フォーム デザイナで必要です。
        InitializeComponent()

        ' InitializeComponent() 呼び出しの後で初期化を追加します。

    End Sub

    Private Sub FirstViewPage_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
        ' ラベルの背景をPictureBoxコントロールに対して透過にする
        mainvisual.Controls.Add(product1)
        mainvisual.Controls.Add(product2)
        mainvisual.Controls.Add(product3)
        mainvisual.Controls.Add(product_body)
        mainvisual.Controls.Add(product_title)
    End Sub

End Class
